### DISCUSSION TITLE
Adding Libraries

### DISCUSSION THREAD

**Discussion by jdarge** [atom/atom#22283] [Q&A]
Hello,

I've never had to manually "install" libraries into the Atom editor before...

Currently, I'm trying to get raylib to work in atom.io; but, I cannot seem to get it to work.

Any ptrs would be greatly appreciated! <3

**Comment by ThatXliner** [comment]
What do you mean by "libraries"?

**Reply by jdarge** [reply]
header files

**Reply by jdarge** [reply]
after downloading raylib...

i tried messing around with gpp-compiler>c compiler options to try and get "raylib.h" to work properly

however it says "raylib.h: No such file or directory

**Reply by ThatXliner** [reply]
This isn't really a problem with Atom but c/c++ itself. Have you tried checking the library's install instructions?

**Comment by jdarge** [comment]
![unknown](https://user-images.githubusercontent.com/79925808/116155708-74e09280-a6b8-11eb-92ca-10245ac972a0.png)

This is what I tried doing. Still looking for a way to resolve this issue

**Reply by icecream17** [reply]
Do you know where raylib.h is in your filesystem?

**Reply by jdarge** [reply]
> 
> 
> Do you know where raylib.h is in your filesystem?

yes i do C:\raylib\raylib\src\raylib.h

**Reply by jdarge** [reply]
i have also tried this (inside of the compiler options)

-IC:\Users\darkr00t\Desktop\New -LC:\raylib\raylib\src -lraylib -lopengl32 -lgdi32

**Reply by jdarge** [reply]
but i still get a fatal error: 

"C:\Users\darkr00t\Desktop\New\main.c:12:20: fatal error: raylib.h: No such file or directory
#include "raylib.h"

I've also tried #include "C:\raylib\raylib\src\raylib.h"

**Reply by icecream17** [reply]
https://stackoverflow.com/questions/35953886/include-path-directory Maybe you can try adding the path to the INCLUDE environment variable? (Also in npm there's a space for params: `-I <file>` instead of `-I<file>`, but this is cpp so maybe i'm silly)
